body/test/test_hello_utils.py

```diff
@@ -90,3 +89,77 @@ def test_get_stretch_directory(self):
             os.environ['HELLO_FLEET_PATH'] = original_fleet_path
         else:
             self.assertEqual(stretch_body.hello_utils.get_stretch_directory(), "/tmp/")
+
```

Comment:
These tests are a good start! LoopStats maintains `loop_rate_avg_hz`, `loop_rate_min_hz`, `loop_rate_max_hz`, `loop_rate_std`, `execution_time_ms`, and `loop_warns`. It would be nice to see individual test cases that ensure the correctness in calculation of each of these parameters. For example, you could write a test case that uses `time.sleep` to create a loop that runs at 5 hz, and check whether `loop_rate_avg_hz` averages out to near 5hz within a reasonable number of iterations.